include/aws/crt/mqtt/Mqtt5Client.h

```diff
@@ -315,6 +318,13 @@ namespace Aws
                  */
                 const Mqtt5ClientOperationStatistics &GetOperationStatistics() noexcept;
 
+                /**
+                 * Create a new connection object from the client5.
+                 *
+                 * @return std::shared_ptr<Crt::Mqtt::MqttConnection>
+                 */
+                std::shared_ptr<Crt::Mqtt::MqttConnection> NewConnection() noexcept;
```

Comment:
Note: We should change the description for MqttConnection as the memory is not going to always be owned by MqttClient anymore.